cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java

```diff
@@ -5,20 +5,20 @@
 @Schema(
         name = "Version Type",
         description = "Version type specifies the type of timeseries response to be received. Can be max aggregate "
-                + "or single version. Max aggregate cannot be run if version date field is specified."
+                + "or single version. Max aggregate cannot be ran if version date field is specified."
```

Comment:
I think this just got missed or mixed up but I'm pretty sure we decide this *should* be `cannot be run`

Reply:
Yes.  Fixed.